modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,139 @@
+
+.Dependencies Version
+[cols="1,1",options="header"]
+|===
+| Dependency | Version
+| ApplicationSet
+| v0.2.0
+| Kustomize
+| v4.2.0
+| Argo CD
+| v2.1.2
+| Helm
+| v3.6.0
+|===
+
+[id="new-features-1-3_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.0:
+
+* After Argo CD is installed, Red Hat Single Sign On (SSO) is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Red Hat SSO installation after the operator is installed which will result in removing the Red Hat SSO deployment from the `openshift-gitops` namespace.
+Additionally, for any new Argo CD instance you can enable Red Hat SSO by setting the flag to true; when the flag is not present, Red Hat SSO will be disabled by default.
+
+* The default Argo CD instance installed by the OpenShift GitOps operator as well as accompanying controllers can now be made to run on the cluster's infrastructure nodes by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
+. Label the existing nodes. In the terminal run the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=""
+----
+If required, you can also apply taints and isolate the workloads on Infrastructure nodes and prevent other workloads from scheduling on these nodes. Run the following command:
++
+[source,terminal]
+----
+$ oc adm taint nodes -l node-role.kubernetes.io/infra
+infra=reserved:NoSchedule infra=reserved:NoExecute
+----
+. Add `runOnInfra` toggle in the `GitOpsService` Custom Resource. Run the following command:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  runOnInfra: true
+----
+. Add `tolerations` to the `GitOpsService` Custom Resource if taints have been added to the nodes, for example:
++
+[source,yaml]
+----
+  spec:
+    runOnInfra: true
+    tolerations:
+    - effect: NoSchedule
+      key: infra
+      value: reserved
+    - effect: NoExecute
+      key: infra
+      value: reserved
+----
+. Verify that the workloads in the `openshift-gitops` namespace are now scheduled on infrastructure nodes by viewing `Pods -> Pod details` for any pod in the console UI.
+
+[NOTE]
+====
+Any manually added `nodeSelectors` and `tolerations` added to the default Argo CD Custom Resource will be overwritten by toggle and `tolerations` in the `GitOpsService Custom Resource`.
+====
+* Internal communications in Argo CD can now be secured using `TLS` and OpenShift cluster certificates. Argo CD routes can now leverage OpenShift cluster certificates in addition to using external certificate managers such as cert-manager.
+* Gain insights into the GitOps environments directy from the DevConsole using the improved environment details view in DevConsole for OCP 4.9.
```

Comment:
From the UI perspective we do not have something called the DevConsole. We only have a **Developer** perspective in the console. So in the docs mention the latter.
'directy from the DevConsole using the improved environment details view in DevConsole for OCP 4.9.' seems to be repeating the console mentions.
Suggest: Use the improved environment details view in the **Developer** perspective of the console 4.9 to gain insights into the GitOps environments./Gain insights into the GitOps environments using the improved environment details view in  the **Developer** perspective of the console 4.9.